src/core/settings/qgssettingsentry.h

```diff
@@ -119,6 +119,20 @@ class CORE_EXPORT QgsSettingsEntryBase
      */
     QString key( const QStringList &dynamicKeyPartList ) const;
 
+    /**
+     * Returns true if the provided key match the settings entry
```

Comment:
could you maybe give a few more words here? 
saying in which scenario you can use it, say that it's useful for dynamic keys.

Reply:
Shouldn't we use capitalized `TRUE` in doc strings?

Reply:
Changed in last commit